bundles/org.openhab.binding.ojelectronics/src/main/resources/ESH-INF/i18n/ojelectronics_de_DE.properties

```diff
@@ -0,0 +1,31 @@
+# binding
+binding.ojelectronics.name = Binding fï¿½r OJElectronics
```

Comment:
There seem to be an issue with the encoding. Umlauts take only two bytes in UTF-8, actually.